### PR TITLE
Rebate cycle

### PR DESCRIPTION
**Pull request by Darruma**

### CODE REVIEW

**Comment by mitche50** [comment]
LGTM - Just need the yaml update to propose and will give approval.

**Review by btcookies — APPROVED** [review]
LGTM

File: .deploy/base/propose_eth_cycle_bot.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
                         - name: propose-eth-cycle-bot
                           image: IMAGE_NAME
                           imagePullPolicy: IfNotPresent
-                          command: ['python', '-m', 'scripts.rewards.eth.propose_eth']
+                          command: ['python', '-m', 'scripts.rewards.rebate.rebate_calculation']
```
Comment (btcookies) [inline .deploy/base/propose_eth_cycle_bot.yaml:20]: shouldn't this be rebate.propose?

**Review by btcookies — CHANGES_REQUESTED** [review]

File: .deploy/base/propose_eth_cycle_bot.yaml (hunk 1 above)
Comment (Darruma) [reply .deploy/base/propose_eth_cycle_bot.yaml:20]: Yep

**Review by btcookies — APPROVED** [review]
LGTM

**Review by mitche50 — APPROVED** [review]

**Pull request merged**